### DISCUSSION TITLE
No module named 'hit'

### DISCUSSION THREAD

**Discussion by my-hawary** [idaholab/moose#20437] [Q&A Getting Started] [answered]
Hi,

I am trying to install MOOSE on my Macbook with M1 following the steps [https://mooseframework.inl.gov/getting_started/installation/conda.html](https://mooseframework.inl.gov/getting_started/installation/conda.html).
For Mambaforge3, i installed MacOSX-arm64. 

When I was trying to do the last step (Compile and test), I got the following error:

ModuleNotFoundError: No module named 'hit'

**Comment by cticenhour** [comment]
Thank you for your message! As this is a question regarding conda installation, I am moving this to the Discussions forum. You might get some email notifications about this.

**Comment by cticenhour** [comment, marked as answer]
To get started in troubleshooting your issue, could you load up your moose conda environment and post the output of the command `conda list`? This is just to confirm that everything we expect to have installed is there. 

EDIT: To summarize the thread below, conda was unable to install the moose packages due to a conda channel that did not exist. Removing the channel allowed proper syncing of the package metadata, which allowed download of the moose packages. See below for more details.

**Reply by my-hawary** [reply]
It seems that the issue is with the packages. There is no packages installed in mambaforge3/envs/moose

When I tried to reinstall the packages using mamba install moose-tools moose-libmesh, I got the following:
RuntimeError: Multi-download failed
public/osx-arm64                                              No change
pkgs/r/osx-arm64                                   279.0 B @ 286.0 B/s 404 failed  0.1s
conda/moose/osx-arm64                              199.0 B @ 203.0 B/s 404 failed  1.0s
conda/moose/noarch                                 195.0 B @ 199.0 B/s 404 failed  1.0s

**Reply by cticenhour** [reply]
Given that these are `404` errors, it appears as though there's an issue either with your internet connection/firewall or the configuration of the conda channels. Could you show the output of the following command? 
```
conda config --show channels
```

**Reply by my-hawary** [reply]
I run the command on the base env. The o/p is:
channels:
  - https://conda.software.inl.gov/public
  - https://mooseframework.org/conda/moose
  - conda-forge
  - defaults

**Reply by cticenhour** [reply]
For comparison, the channel configuration for a newly configured M1 Mac Mini is as follows:
```
% conda config --show channels
% channels:
    - https://conda.software.inl.gov/public
    - conda-forge
  ```

the presence of `conda/moose` in your output is odd to me, as I only see `public` or `conda-forge` in the channel sync and download process output.

**Reply by cticenhour** [reply]
The second channel you have listed does not exist, which explains the 404 errors. Please remove it by performing the following command:
```
conda config --remove channels https://mooseframework.org/conda/moose
```
And then try installing the packages again in the activated moose environment.

**Reply by my-hawary** [reply]
It seemed to be working fine so far. I am running a test right now. I will let you now if I got any issues.
Thank you so much.
